src/VisualStudio/Core/Def/LanguageService/AbstractPackage.cs

```diff
@@ -27,18 +27,13 @@ internal IComponentModel ComponentModel
 
     protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
     {
-        await base.InitializeAsync(cancellationToken, progress).ConfigureAwait(true);
-
-        await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
-
-        _componentModel_doNotAccessDirectly = (IComponentModel?)await GetServiceAsync(typeof(SComponentModel)).ConfigureAwait(true);
+        _componentModel_doNotAccessDirectly = (IComponentModel?)await GetServiceAsync(typeof(SComponentModel)).ConfigureAwait(false);
         Assumes.Present(_componentModel_doNotAccessDirectly);
+
```

Comment:
nit: feel free to do in a follow up
```suggestion
```

Reply:
whoops! I had a whole bunch of logging that I was using for local verification, and looks like I didn't remove the newline that I had next to it.